ludwig/features/audio_feature.py

```diff
@@ -214,7 +216,7 @@ def reduce(series):
             merged_stats["cropped"],
             audio_file_length_limit_in_s,
         )
-        logger.debug(print_statistics)
+        logging.debug(print_statistics)
```

Comment:
Why not use logger?

Reply:
Oh, I think we are cleaning up the use of `logger` in favor of `logging`. Open issue: https://github.com/ludwig-ai/ludwig/issues/2045

Reply:
Ah, gotcha!